### PR TITLE
Adds identify method to GoogleAnalytics adapter.

### PR DESCRIPTION
**Pull request by denneralex**
Hey, I noticed that GoogleAnalytics adapter was missing the identify method.
This is my suggestion to add that.

### CODE REVIEW

File: tests/unit/metrics-adapters/google-analytics-test.js [hunk 1]
```diff
@@ -15,6 +15,17 @@ moduleFor('ember-metrics@metrics-adapter:google-analytics', 'google-analytics ad
   }
 });
 
+test('#identify calls ga with the right arguments', function(assert) {
+  var adapter = this.subject({ config });
```
Comment (poteto) [inline tests/unit/metrics-adapters/google-analytics-test.js]: We should use `const` over `var` here

**Comment by poteto** [comment]
Apart from the one comment, LGTM

**Comment by denneralex** [comment]
I just changed var to const as you asked.

**Comment by poteto** [comment]
Awesome, thank you!

**Pull request merged**